### PR TITLE
[TASKMGR] Fix UI on SwitchTo for multiple application

### PR DESCRIPTION
**Pull request by KRosUser**
Fix discrepancy on "Switch To" action when multiple applications are selected
image

Menu is available (and only Switch to one of the selected apps) but button is disabled.
WinXPSP3 : both active and behaves like ReactOS menu (only Switch to one of the selected apps)
image

Proposed changes : Make button and menu consistent

### CODE REVIEW

**Comment by HBelusca** [comment]
Actually this is amusing but Microsoft fixed this idiosyncrasy in the Windows 7+ (and maybe Vista) Taskmgr, where now, selecting more than 1 task disables the "Switch To" button.
https://www.howtogeek.com/wp-content/uploads/2012/09/image86.png
image

**Comment by KRosUser** [comment]
> Actually this is amusing but Microsoft fixed this idiosyncrasy in the Windows 7+ (and maybe Vista) Taskmgr, where now, selecting more than 1 task disables the "Switch To" button. So please fix our taskmgr by instead implementing that windows 7 behaviour (which was the intended one in ROS). https://www.howtogeek.com/wp-content/uploads/2012/09/image86.png image

OK, i'll do it this way

**Comment by KRosUser** [comment]
@HBelusca : Implemented as requested !

File: base/applications/taskmgr/applpage.c [hunk 1]
```diff
@@ -512,14 +512,14 @@ void ApplicationPageUpdate(void)
...
+    /* Enable "Switch To" button only if only one app is selected */
     if (ListView_GetSelectedCount(hApplicationPageListCtrl) == 1 )
     {
         EnableWindow(hApplicationPageSwitchToButton, TRUE);
     }
     else
     {
-    EnableWindow(hApplicationPageSwitchToButton, FALSE);
+        EnableWindow(hApplicationPageSwitchToButton, FALSE);
     }
```
Comment (learn-more) [inline base/applications/taskmgr/applpage.c:523]: Why not replace this entire block with just:
`EnableWindow(hApplicationPageSwitchToButton, ListView_GetSelectedCount(hApplicationPageListCtrl) == 1);`

Comment (KRosUser) [reply base/applications/taskmgr/applpage.c:523]: Yes good idea !

**Review by HBelusca — APPROVED** [review]

**Pull request merged**